PRIVACY.md

```diff
@@ -0,0 +1,48 @@
+
+For the purposes of this notice, WE refers to the entity of Flagrow and its proprietors, FORUM refers to a Flarum website that has installed Flagrow Bazaar and YOU refers to its owner, APPLICATION refers to Flagrow Bazaar, WEBSITE refers to flagrow.io. 
+
+## Privacy Notice
+
+This privacy notice discloses the privacy practices for the APPLICATION. This privacy notice applies solely to information collected by this application and its associated WEBSITE. It will notify of the following:
+
+* What personally identifiable information is collected from YOU through the WEBSITE, how it is used and with whom it may be shared.
+* What choices are available to YOU regarding the use of your data.
+* The security procedures in place to protect the misuse of information YOU provide.
+* How YOU can correct any inaccuracies in the information.
+
+### Information Collection, Use, and Sharing
+WE are the sole owners of the information collected by the APPLICATION during the course of normal operation. 
+
+If YOU choose to connect the APPLICATION to the WEBSITE, then WE also own any information provided. Unless YOU ask us not to, we may contact YOU via email in the future to tell you about specials, new products or services, or changes to this privacy policy.
+
+If YOU choose to email us or directly contact us in another way, WE also have access to/collect information that YOU voluntarily provide. WE will also use this information to respond to YOU, regarding the reason YOU contacted us. WE will not share your information with any third party outside of our organization, other than as necessary to fulfill your request, e.g. to provide a service.
+
+**WE will not sell or rent this information to anyone for any purposes.**
+
+### What We Collect
+
+To provide the best service for this APPLICATION, WE may collect the following from YOU. 
+
+* The contents of the Flarum configuration file of your FORUM, except for service authentication credentials (e.g. a database).
+* The contents of files relating to Composer configuration of your FORUM.
+* Preferences marked in the APPLICATION, such as favorites.
+* The IP and url of your FORUM.
+* An email address YOU provide  (if connected to the WEBSITE).
+* A name YOU provide (if connected to the WEBSITE).
+* Information that connects YOU to your FORUM.
+* Other statistical information related to your FORUM. 
+
+### Your Access to and Control Over Information
+YOU may opt out of any future contacts from us at any time. YOU can do the following at any time by contacting us via email at privacy@flagrow.io:
```

Comment:
Weird phrasing.
"You may opt out of any future communication from us at any time. You can do so by emailing us at privacy@flagrow.io"